openerp/addons/base/i18n/base.pot

```diff
@@ -10041,6 +10299,18 @@ msgid "Missing SMTP Server"
 msgstr ""
 
 #. module: base
+#: code:addons/models.py:6036
+#, python-format
+msgid "Missing required value for the field '%s' (%s)"
```

Comment:
Actually we have these `%s` in many [other strings](https://github.com/odoo/odoo/commit/d780f9476d43eb4fd25c852dd4e691002f4a6dcf#diff-eedcb99c2a3b66f9db796c8b09a30bcaR142) so I think translators are used to the meaning of it (it's even recognised by transifex and gives you a warning if you remove it in your translation) and I don't want to get people translating `%(field)s`by `%(champ)s` :wink: